src/test/groovy/edu/ie3/simona/model/participant/PvModelIT.groovy

```diff
@@ -55,8 +57,8 @@ class PvModelIT extends Specification implements PvModelITHelper {
     setDefault(US)
 
     pvModels = createPvModels()
-    weatherMap = getWeatherData()
-    resultsMap = getResultsData()
+    weatherMap = weatherData
+    resultsMap = resultsData
```

Comment:
Honestly, I hate that groovy allows this and Codacy enforces this. But we were already halfway there anyway, so at least let's commit fully